package.json

```diff
@@ -1,56 +1,61 @@
 {
   "name": "visearch-javascript-sdk",
-  "version": "3.1.4",
+  "version": "4.0.0-rc.0",
   "description": "JavaScript SDK for ViSearch of visenze.com",
-  "main": "./js/index.js",
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./esm/index.js",
+      "require": "./cjs/index.js",
+      "default": "./cjs/index.js"
+    }
+  },
   "scripts": {
-    "eslint": "eslint *.js js/*.js",
-    "gulp": "gulp",
+    "lint": "eslint src --ext .ts",
```

Comment:
I added in lint for types.

For test I don't think it's really necessary until the tests are in ts, otherwise you will need separate lint rules for js/ts

Reply:
Alright sure, we can deal with it later